### DISCUSSION TITLE
Possible to chain search engines?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [gdh1995/vimium-c#1056] [Q&A] [answered]
Wondering whether it's possible. I'd like to do something along the lines of `v.c g <query>` to copy a Google search (filled with query) URL to clipboard.

**Comment by gdh1995** [comment, marked as answer]
A recommended way is just to write your own "search engine":
```
v.g: vimium://copy/https://www.google.com/search?ie=utf-8&q=$s re= Copy Google Search URL
```

**Reply by limaceous-bushwhacker** [reply]
Ty, guess this'll have to do meanwhile.
